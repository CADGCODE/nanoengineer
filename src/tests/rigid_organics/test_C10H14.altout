======= test_C10H14.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= stderr =======
cutover to fine tolerance at 2
== structure comparison ==
error reading atom 0 from test_C10H14.xyzcmp
could not read base positions file from -B<filename>
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_C10H14.mmp  > test_C10H14.xyz
final RMS gradient=2183.400309 after 5 iterations
== structure comparison ==
======= test_C10H14.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_C10H14.mmp
# Output File: test_C10H14.xyz
# Trace File: test_C10H14.trc
# Number of Atoms: 26
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#
0 2902.76 4298.00
1 2206.99 2668.16
2 2183.80 2627.68
3 2183.41 2626.99
4 2183.40 2626.98
5 2183.40 2626.98
# Done: Minimization final rms: 2183.400309, highForce: 2626.982633
======= test_C10H14.xyz =======
26
RMS=2183.400309
C -1.600885 0.659267 0.422428
C -1.410870 -0.888297 0.415566
C -0.267574 1.355592 0.010609
C -0.286908 -1.275688 1.424509
C -1.003831 -1.350073 -1.013830
C 0.141879 0.888378 -1.416012
C 0.853523 0.960070 1.019823
C 1.049520 -0.586955 1.008517
C 1.451204 -1.056774 -0.417304
C 0.331971 -0.659383 -1.427511
H -2.401477 0.938165 -0.276320
H -1.890988 0.987872 1.429562
H -2.347100 -1.373294 0.703938
H -0.400957 2.440299 0.016105
H -0.156150 -2.369446 1.435628
H -0.572855 -0.944408 2.434674
H -0.875965 -2.443569 -1.027420
H -1.797215 -1.078725 -1.724370
H 1.080764 1.380255 -1.705487
H -0.639882 1.170555 -2.136241
H 1.794134 1.458004 0.744456
H 0.571957 1.284781 2.034489
H 1.835830 -0.860917 1.713302
H 2.402404 -0.586103 -0.708647
H 1.579285 -2.150185 -0.420365
H 0.613186 -0.983421 -2.432097
