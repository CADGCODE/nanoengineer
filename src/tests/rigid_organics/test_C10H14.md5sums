stdout E4D82E0EB8F95EC72F191A5499719A12
stderr D41D8CD98F00B204E9800998ECF8427E
test_C10H14.xyz 0B056CA22016567AF35DDA1259568D70
