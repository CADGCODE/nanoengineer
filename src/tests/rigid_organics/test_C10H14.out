======= test_C10H14.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= stderr =======
cutover to fine tolerance at 2
== structure comparison ==
error reading atom 0 from test_C10H14.xyzcmp
could not read base positions file from -B<filename>
